recipes/dataframe/all/conanfile.py

```diff
@@ -50,7 +50,12 @@ class DataFrameConan(ConanFile):
 
     @property
     def _min_cppstd(self):
-        return "20" if Version(self.version) >= "2.1.0" else "17"
+        if Version(self.version) < "2.1.0":
+            return "17"
+        elif Version(self.version) <= "2.2.0":
+            return "20"
+        else:
+            return "23"
```

Comment:
Im going to block merging this until we support C++23 in CCI, I'll push for it next week, thanks a lot for your patience

(Else no binaries for the new version will be generated)